### ISSUE TITLE
Airport Revival: London Heathrow (EGLL)

### ISSUE THREAD

**Issue by jakcharvat** [openscope/openscope#985]
Reintroduce London Heathrow.

**Comment by jdouglas77** [comment]
is it done yet?

**Comment by Fechulo** [comment]
@jdouglas77 Adding an airport to the sim takes a lot of work and time from several people. @jakcharvat has been working very hard on this airport, but it will be a while until it is added back into openScope. For the moment, you can look at the deployment for this airport [here](https://openscope-dev-pr-986.herokuapp.com/), but bear in mind that the airport may not be stable at this point, so expect errors.

**Comment by jdouglas77** [comment]
Weirdly enough the OMDB airport works through the equivalent link however EGLL doesn't even show up aha

**Comment by indianbhaji** [comment]
Yeah, because you’ve commented on the issue for Dubai. That link only updates with the side branch we have set up for Dubai only. @jdouglas77

**Comment by indianbhaji** [comment]
@jdouglas77 Wrong link, that one links to the branch for Dubai. https://openscope-dev-pr-986.herokuapp.com/ for Heathrow

**Comment by jdouglas77** [comment]
yep i realise that but when i click on egll it won't load, anyway not a big issue, ill just wait till its released :D

**Comment by jakcharvat** [comment]
@jdouglas77 sorry for not replying earlier, I was off on a vacation, got back today so I'll continue working on it. I'd say I'm about 50% done, hopefully can have it in the son soon
